DiscordBot/Commands/EventCommands.cs

```diff
@@ -279,20 +279,24 @@ public async Task ListEvents(CommandContext context)
         {
             var eventsList = await eventsSheetsService.ListEventsAsync();
 
+            var sortedEventsList = eventsList
+                .Where(x => x.Time > DateTime.Now)
```

Comment:
Might it be better to show events that happened earlier on the same day?

Reply:
No, I think all upcoming events is the most useful. It's always possible to add a separate command for listing all events at some point if necessary 🙂